src/main/java/spoon/experimental/SpoonifierVisitor.java

```diff
@@ -143,6 +144,11 @@ && isLeafTypeReference(element.getParent())) {
 			element.accept(propertyScanner);
 		}
 
+		if (element instanceof CtType<?> && !((CtType<?>) element).isAnonymous()) {
+			// anonymous classes don't have a package
+			result.append(printTabs() + variableName + ".setParent(factory.Package().getOrCreate(\"" + ((CtType<?>) element).getPackage().getQualifiedName() + "\"));\n");
```

Comment:
Qodana complains below. Would it go away if I fix this line or I will have to fix the entire file?